_source/logzio_collections/_log-sources/microsoft-graph.md

```diff
@@ -72,8 +72,8 @@ please [open an issue](https://github.com/logzio/microsoft-graph/issues/new) at
 
     Click **Grant admin consent for Default Directory > Yes**.
 
-    If the _Grant admin consent for Default Directory_ button is disabled,
-    contact an Azure administrator for your organization.
+    Only Azure administrators can change the _Grant admin consent for Default Directory_ setting.
```

Comment:
Perhaps 'only azure admins can grant consent for default directory...?

Reply:
Revised, please take a look

Reply:
Love it